views/support/show.ejs

```diff
@@ -1,93 +1,104 @@
         </div>
     </div>
-    <hr>
-    <div class="row">
-        <div class="col-sm-12 col-md-6">
-            <form action="/support/<%=complaint._id%>?_method=PUT" method="POST">
-                <div class="form-group">
-                    <label for="" class="ticket-details">Select Status:</label>
-                    <select class="form-control" name="status" id="statusSelect">
-                      <option value="">--Select Status--</option>  
-                      <option value="Open">Open</option>
-                      <option value="Waiting List">Waiting List</option>
-                      <option value="Closed">Closed</option>
-                    </select>
-                    <p id="status-warning"><em>*Note: Submitting the status will automatically notify the user</em></p>
-                  </div>
-                  <div class="form-group">
-                    <input type="submit" value="Submit" class="btn btn-sm btn-success">
-                 </div>
-              </form>
-        </div>
-        <div class="col-sm-12 col-md-6">
-            <form action="/support/<%=complaint._id%>?_method=PUT" method="POST">
-                <div class="form-group">
-                   <label for="" class="ticket-details">Assign To:</label>
-                   <select class="form-control" name="assignedTo" id="statusSelect">
-                     <option value="">--Assign To--</option>  
-                     <option value="Open">John Doe</option>
-                     <option value="Waiting List">Iron Man</option>
-                     <option value="Closed">Aladeen</option>
-                   </select>
+    <div class="col-sm-12 col-md-8">
+        <div class="col-12" id="user-info">
+            <h2 class="user-details" style="text-align: left;">Ticket Information:</h2>
+            <hr class="style-one" />
+            <div class="details">
+                <div style="float: left;">
+                    <ul>
+                        <li><strong class="ticket-details">Title/Subject:</strong></li>
+                        <li><strong class="ticket-details">Created At:</strong></li>
+                        <li><strong class="ticket-details">Opened At:</strong></li>
+                        <li><strong class="ticket-details">Updated At:</strong></li>
+                        <li><strong class="ticket-details">Ticket ID/Ref. No:</strong></li>
+                        <li><strong class="ticket-details">Status:</strong></li>
+                    </ul>
                 </div>
-                <div class="form-group">
-                   <input type="submit" value="Submit" class="btn btn-sm btn-success">
+                <div style="float: right;">
+                    <ul>
+                        <li><%=complaint.name%></li>
+                        <li><%= new Date(complaint.createdAt).toLocaleDateString("en")%></li>
+                        <li><%= new Date(complaint.reviewStartedAt).toLocaleDateString("en")%></li>
```

Comment:
new Date then into it pass value from DB and show it in more human friendly way.